roboteam_ai/src/world/Field.cpp

```diff
@@ -94,84 +96,87 @@ std::vector<std::pair<Vector2, Vector2>> Field::getBlockadesMappedToGoal(bool ou
             //object completely faced the wrong way
-            if (point1val<=0 && point2val<=0){
-                validObstacle=false;
+            if (point1val <= 0 && point2val <= 0) {
+                validObstacle = false;
             }
                 //these following 2 cases are identical in logic but mirrored; one point hits the backline, other does not.
                 // in that case, we pick the appropriate goalPost which would be right for the obstacle as new Point and check if this interval is valid
-            else if(point1val<=0&&point2val>0){
-                validObstacle=true;
-                if (point1.y<point2.y){
-                    point1=upperGoalSide;
+            else if (point1val <= 0 && point2val > 0) {
+                validObstacle = true;
+                if (point1.y < point2.y) {
+                    point1 = upperGoalSide;
                 }
-                else{
-                    point1=lowerGoalSide;
+                else {
+                    point1 = lowerGoalSide;
                 }
             }
-            else if(point2val<=0&&point1val>0){
-                validObstacle=true;
-                if (point2.y<point1.y ){
-                    point2=upperGoalSide;
+            else if (point2val <= 0 && point1val > 0) {
+                validObstacle = true;
+                if (point2.y < point1.y) {
+                    point2 = upperGoalSide;
                 }
-                else{
-                    point2=lowerGoalSide;
+                else {
+                    point2 = lowerGoalSide;
                 }
             }
-            else{
+            else {
                 //'normal' obstacle; check if the points are at good points
-                validObstacle=true;
+                validObstacle = true;
             }
 
             // check if both points are below or above the goal (this invalidates it again)
-            if (validObstacle){
+            if (validObstacle) {
                 bool bothPointsBelowGoal = point1.y <= lowerGoalSide.y && point2.y <= lowerGoalSide.y;
                 bool bothPointAboveGoal = point1.y >= upperGoalSide.y && point2.y >= upperGoalSide.y;
-                if (bothPointsBelowGoal||bothPointAboveGoal){
-                    validObstacle=false;
+                if (bothPointsBelowGoal || bothPointAboveGoal) {
+                    validObstacle = false;
                 }
             }
-            if (validObstacle ) {
+            if (validObstacle) {
                 // constrain the blockades to within the goal
                 if (point1.y > point2.y) { // point1 is largest
                     point1.y = std::min(point1.y, upperGoalSide.y);
                     point2.y = std::max(point2.y, lowerGoalSide.y);
-                    blockades.emplace_back(std::make_pair(point2,point1)); // the first element in the pair is the smallest
-                } else { // point2 is largest
+                    blockades.emplace_back(
```

Comment:
yeah just make this on one line and put the comment somewhere else if you really care about ctrl alt L that much.